docker/Dockerfile

```diff
@@ -30,4 +30,4 @@ RUN chown -R notify:notify /home/vcap/app \
 
 USER notify
 
-ENTRYPOINT [ "/home/vcap/app/entrypoint.sh" ]
+#ENTRYPOINT [ "/home/vcap/app/entrypoint.sh" ]
```

Comment:
If we are going to use this dockerfile for both local development and also in production, it does make sense to get rid of this line. This will mean we can pass a much wider range of commands into our dockerfile to run.